File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multitype/FavoriteArtistViewHolder.java
package com.thoughtbot.expandablerecyclerview.sample.multitype;

import android.view.View;
import android.widget.TextView;
import com.thoughtbot.expandablerecyclerview.sample.R;
import com.thoughtbot.expandablerecyclerview.viewholders.ChildViewHolder;

public class FavoriteArtistViewHolder extends ChildViewHolder {

  private TextView favoriteArtistName;

  public FavoriteArtistViewHolder(View itemView) {
    super(itemView);
    favoriteArtistName = (TextView) itemView.findViewById(R.id.list_item_favorite_artist_name);
  }

  public void setArtistName(String name) {
    favoriteArtistName.setText(name);
  }

}

File: sample/src/main/java/com/thoughtbot/expandablerecyclerview/sample/multitype/FavoriteArtistViewHolder.kt
package com.thoughtbot.expandablerecyclerview.sample.multitype

import android.view.View
import android.widget.TextView
import com.thoughtbot.expandablerecyclerview.sample.R
import com.thoughtbot.expandablerecyclerview.viewholders.ChildViewHolder

class FavoriteArtistViewHolder(itemView: View) : ChildViewHolder(itemView) {
    private val favoriteArtistName: TextView
    fun setArtistName(name: String?) {
        favoriteArtistName.text = name
    }

    init {
        favoriteArtistName = itemView.findViewById<View>(R.id.list_item_favorite_artist_name) as TextView
    }
}